Dockerfile

```diff
@@ -3,19 +3,22 @@
 FROM alpine
 
 # Enable HTTPS support in wget and set nsswitch.conf to make resolution work within containers
-RUN apk add --no-cache --update openssl \
+RUN : \
```

Comment:
Please don't submit unrelated formatting changes in a version update. I don't like this style very much, to be honest, and this now prevents me from merging the actual update, which I do like.

Reply:
I thought you might like it. Amended in adb72e8a1e40ab78b3551f8d0ac3f6d423bd7e97